src/prune/tooling/artillery/artillery.cpp

```
@@ -0,0 +1,40 @@
+#include "imgui.h"
+
+#include "prune/tooling/artillery/artillery.hpp"
+#include "prune/tooling/imgui/layout.hpp"
+#include "prune/tooling/imgui/property_table.hpp"
+
+namespace prune {
+
+    namespace {
+        [[nodiscard]] const char* turn_label(ArtilleryTurn turn) noexcept
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)